src/material-table.js

```diff
@@ -182,42 +182,23 @@ export default class MaterialTable extends React.Component {
 
     calculatedProps.actions = [...(calculatedProps.actions || [])];
 
-    if (calculatedProps.options.selection)
```

Comment:
This if have been merged because only two lines differ
```
              return { action: action, position: "toolbarOnSelect" };
            else return { ...action, position: "toolbarOnSelect" };
```
```
              return { action: action, position: "row" };
            else return { ...action, position: "row" };
```